scripts/run-dependencies.sh

```diff
@@ -17,7 +19,7 @@ docker run --rm -d \
   --network=$NETWORK \
   --name=$API \
   -p 8888:8888 \
-  mrcide/hintr:mrc-418-rgf
+  mrcide/hintr:latest
```

Comment:
Would like to also use the configured hintr version here but at the moment there is no image tagged with "master" being pushed to dockerhub (https://dideimperial.slack.com/archives/GL5N84ZFU/p1566896766068600). We should change this once there is. Similarly have added a PR for tagging hint-db with "master": https://github.com/mrc-ide/hint-db/pull/3